vectortiles/backends/postgis/functions.py

```diff
@@ -28,3 +28,10 @@ class AsMVTGeom(GeoFunc):
     @cached_property
     def output_field(self):
         return RawGeometryField()
+
+class Force2D(GeoFunc):
+    function = "ST_FORCE2D"
+
+    @cached_property
+    def output_field(self):
+        return RawGeometryField()
```

Comment:
If you try to use GeomOutputGeoFunc instead of GeoFUnc, is this cached_property really usefull ?